### PR TITLE
Update documentation to indicate support for Microdata and requirements

### PR DESCRIPTION
**Pull request by sohanirao**

### CODE REVIEW

File: docs/page-markup.md [hunk 1]
```diff
@@ -51,5 +51,38 @@ Thus, notice that:
 
 ## Microdata markup
 
-Microdata format support is coming soon.
+SWG uses schema.org markup. Using Microdata, the mark up could look like this:
```
Comment (dvoytenko) [inline docs/page-markup.md]: 1. SWG -> SwG
2. schema.org -> Schema.org

File: docs/page-markup.md [hunk 2]
```diff
@@ -51,5 +51,38 @@ Thus, notice that:
...
 
+```
+<div itemscope itemtype="http://schema.org/NewsArticle http://schema.org/Other" id="top">
```
Comment (dvoytenko) [inline docs/page-markup.md]: Let's remove "http://schema.org/Other" and `id="top"` here.

File: docs/page-markup.md [hunk 3]
```diff
@@ -51,5 +51,38 @@ Thus, notice that:
...
+  <meta itemprop="isAccessibleForFree" content="false"/>
+  <div itemprop="isPartOf" itemscope itemtype="http://schema.org/CreativeWork http://schema.org/Product">
+    <meta itemprop="name" content="New York Times"/>
```
Comment (dvoytenko) [inline docs/page-markup.md]: Please use norcal_tribune values everywhere. See LD example above.

File: docs/page-markup.md [hunk 4]
```diff
@@ -51,5 +51,38 @@ Thus, notice that:
...
+    <meta itemprop="productID" content="pub1:premium"/>
+  </div>
+  <div itemprop="articleBody" class="paywalled-section">
```
Comment (dvoytenko) [inline docs/page-markup.md]: Remove this one.

File: docs/page-markup.md [hunk 5]
```diff
@@ -51,5 +51,38 @@ Thus, notice that:
...
+    Paid content possibly.
+  </div>
+  <div itemscope itemtype="http://schema.org/OtherSection" id="bottom">
```
Comment (dvoytenko) [inline docs/page-markup.md]: Ditto: remove this one es well.

File: docs/page-markup.md [hunk 6]
```diff
@@ -51,5 +51,38 @@ Thus, notice that:
...
+    <meta itemprop="isAccessibleForFree" content="True"/>
+  </div>
+</div>
+```
+
+The requirements to retreive a usable configuration from embedded microdata are:
+ 1. Presence of atleast one div marked up as itemscope with itemtype containing 'NewsArticle' in
```
Comment (dvoytenko) [inline docs/page-markup.md]: 1. "at least"
2. "one div" -> "one element"

File: docs/page-markup.md [hunk 7]
```diff
@@ -51,5 +51,38 @@ Thus, notice that:
...
+    schema vocabulary (indicated by URL http://schema.org/)
+ 2. With presence of such a node as parent, a valid configuration is returned if it
+   - contains a meta in one of its childres, with item property "isAccessibleForFree" indicating
+     access info that is not broken in itemscope ownership upto the above parent and
+   - contains a child itemscope of type containing 'Product' in schema vocabulary and a item
+     property 'productID' that is not broken in itemscope ownership upto the above parent
+
+In the example above, the div with id = "bottom" is ignored. The result configuration will be as
+follows:
+  isAccessibleForFree: false, productID: pub1:premium
```
Comment (dvoytenko) [inline docs/page-markup.md]: Ditto: "norcal" example

File: docs/page-markup.md [hunk 8]
```diff
@@ -51,5 +51,38 @@ Thus, notice that:
...
+  <div itemscope itemtype="http://schema.org/OtherSection" id="bottom">
+    <meta itemprop="isAccessibleForFree" content="True"/>
+  </div>
+</div>
+```
+
+The requirements to retreive a usable configuration from embedded microdata are:
+ 1. Presence of atleast one div marked up as itemscope with itemtype containing 'NewsArticle' in
+    schema vocabulary (indicated by URL http://schema.org/)
+ 2. With presence of such a node as parent, a valid configuration is returned if it
```
Comment (dvoytenko) [inline docs/page-markup.md]: We should reduce as people familiar with microdata will know what this means. E.g. "NewsArticle can specify productID within a sub-itemtype of Product, and isAccessibleForFree". The details of validation shouldn't be in the docs.

Comment (sohanirao) [reply docs/page-markup.md]: Done

File: docs/page-markup.md (hunk 2 above)
Comment (sohanirao) [reply docs/page-markup.md]: Done.

File: docs/page-markup.md (hunk 5 above)
Comment (sohanirao) [reply docs/page-markup.md]: Done.

File: docs/page-markup.md (hunk 4 above)
Comment (sohanirao) [reply docs/page-markup.md]: Done.

File: docs/page-markup.md (hunk 1 above)
Comment (sohanirao) [reply docs/page-markup.md]: Done

**Pull request merged**